examples/with-rollup/turbo.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "https://turborepo.org/schema.json",
+  "pipeline": {
+    "build": {
+      "outputs": [".next/**"]
```

Comment:
do we need to add `outputs`  for `ui` package?  that would help illustrate how turbo works across multiple workspaces